app/services/typicode.js

```diff
@@ -0,0 +1,27 @@
+const request = require('request-promise'),
+  typicodePath = 'https://jsonplaceholder.typicode.com',
+  apiError = require('../errors');
+
+function options(endpoint) {
+  return {
+    uri: endpoint,
+    json: true,
+    resolveWithFullResponse: false
+  };
+}
+
+exports.requestAlbums = () => {
+  try {
+    return request(options(`${typicodePath}/albums`));
+  } catch (error) {
+    throw apiError(error.message);
+  }
+};
+
+exports.requestAlbumPhotos = albumId => {
+  try {
+    return request(options(`${typicodePath}/photos?albumId=${albumId}`));
+  } catch (error) {
+    throw apiError(error.message);
```

Comment:
Ditto with the try catch.

Reply:
By try catch I mean this comment https://github.com/wolox-training/lm-express-js/pull/2#discussion_r288315190